templates/ffc-template-node/template.yaml

```diff
@@ -0,0 +1,158 @@
+          description: Describe what this component does
+
+        system:
+          title: System
+          type: string
+          description: Select the system that this component will be a part of
+          ui:field: EntityPicker
+          ui:options:
+            allowArbitraryValues: false
+            catalogFilter:
+              - kind: System
+
+        component_type:
+          title: Component Type
+          type: string
+          description: Select the type of this component. Allowed values - Service (a backend service or API), Web (a web application), Library (a library or package that can be consumed by other applications)
+          default: service
+          enum:
+            - service
+            - website
+            - library
+          enumNames:
+            - Service
+            - Web
+            - Library
+
+        component_lifecycle:
+          title: Component Lifecycle
+          type: string
+          description: Select this component's lifecycle. These are aligned to GDS project phases.
+          default: alpha
+          enum:
+            - discovery
+            - alpha
+            - beta
+            - live
+            - retirement
+          enumNames:
+            - Discovery
+            - Alpha
+            - Beta
+            - Live
+            - Retirement
+
+        owner:
+          title: Owner
+          type: string
+          description: Select the team who will own this component
+          ui:field: OwnerPicker
+          ui:options:
+            catalogFilter:
+              - kind: Group
+                spec.type: team
+
+    - title: Git Repository
+      required:
+        - repoUrl
+      properties:
+        repoUrl:
+          title: Repository Location
+          type: string
+          ui:field: RepoUrlPicker
+          ui:options:
+            allowedHosts:
+              - github.com
+            allowedOrganizations:
+              - defra-adp-sandpit
+            requestUserCredentials:
+              secretsKey: USER_OAUTH_TOKEN
+              additionalScopes:
+                github:
+                  - workflow
+
+  steps:
+    - id: template
+      name: Fetch Skeleton + Template
+      action: fetch:template
+      input:
+        url: ./skeleton
+        values:
+          component_id: ${{ parameters.component_id }}
+          project_name: ${{ parameters.component_id }}
+          description: ${{ parameters.description }}
+          owner: ${{ parameters.owner }}
+          workstream: ${{ parameters.owner | parseEntityRef | pick('name') }}
+          repo_url: ${{ parameters.repoUrl }}
+          repo_project_slug: ${{ parameters.repoUrl | projectSlug }}
+          component_type: ${{ parameters.component_type }}
+          component_lifecycle: ${{ parameters.component_lifecycle }}
+          system: ${{ parameters.system }}
+
+    - id: publish
+      name: Publish
+      action: publish:github
+      input:
+        allowedHosts: ["github.com"]
```

Comment:
These settings look good. @DaRoszko, can we review them tomorrow with James?

Reply:
yes